_data/contents.yml

```diff
@@ -39,6 +39,9 @@ specifics:
   - ets
   - mnesia
   - debugging
+ecto:
```

Comment:
@SophieDeBenedetto now that you're adding new lessons I thought you might like to know that this is the file we use to manage the order of our lessons.

This way we don't need to prefix filenames with a number (like `03_associations.md`).

Reply:
awesome, I would like to know that 😃

Reply:
HA I never knew this and suggested the aforementioned naming convention. My bad! Great to know.